docs/tab-widgets/distributed-trace-receive.asciidoc

```diff
@@ -117,6 +117,35 @@ agent.startTransaction('my-service-b-transaction', { childOf: traceparent }) <2>
 // ***************************************************
 // ***************************************************
 
+// tag::php[]
+
+1. Receive the distributed tracing context on the server side and unserialize it back to a new `DistributedTracingData` object.
+
+2. Begin a new transaction using the agent's public API. For example, use {apm-php-ref-v}/public-api.html#api-elasticapm-class-begin-current-transaction[`ElasticApm::beginCurrentTransaction`]
+and pass the new `DistributedTracingData` object as a parameter.
+This will create a new transaction or span as a child of the incoming trace context.
```

Comment:
It's not entirely correct - as name implies `beginCurrentTransaction` is used to create a new transaction and not a span. I saw some other agents say something similar in this document (i.e., that distributed tracing context on the receiving side can be used to create a new span and not only transaction). I am not sure if it's correct for those agents or not - it might worth making sure with the relevant teams. But for PHP agent distributed tracing context on the receiving side can be passed to create a new transaction but not a new span. A new spans can then be created as children of the new transaction (and the distributed tracing context will passed to them automatically by the new transaction). The confusion might be stemming from the fact that on the sending side the source of the distributed tracing context (that then is passed to the receiving side) can be either transaction or span whatever is the current entity at the point `getDistributedTracingData` is invoked. That source of the distributed tracing context will then be considered to be the parent of the new transaction on the receiving side.

Reply:
> I am not sure if it's correct for those agents or not - it might be worth making sure with the relevant teams.

Thanks. The agent devs created that content for the most part, but I'll double-check just to be safe.

> The confusion might be stemming from the fact that on the sending side the source of the distributed tracing context (that then is passed to the receiving side) can be either transaction or span whatever is the current entity at the point getDistributedTracingData is invoked.

Yes, that's very helpful. I'll fix.

Reply:
Addressed in c9b17b5. Will follow up with the other agents in a separate PR.